code/modules/jobs/job_types/other/merc_classes/enforcer.dm

```diff
@@ -0,0 +1,77 @@
+/datum/job/advclass/mercenary/enforcer
+	title = "Enforcer"
+	tutorial = "You're an exiled enforcer that took refuges in the valorian regions long ago, near the beginning of Z's ascension, robed in black, and known for wild antics, loose camaraderie and a huge hatred for dark elves and the descendants of Zizo, You once used your blade to shake down anyone who hasn't paid their 'protection fees', nowadays, you will fight for anyone for the right price."
+	allowed_races = list(SPEC_ID_ELF, SPEC_ID_HUMEN, SPEC_ID_HALF_ELF)
+	outfit = /datum/outfit/mercenary/enforcer
+	category_tags = list(CTAG_MERCENARY)
+	total_positions = 5
+	cmode_music = 'sound/music/cmode/Combat_Weird.ogg'
+
+	jobstats = list(
+		STATKEY_CON = 3,
+		STATKEY_END = 2,
+		STATKEY_STR = 1,
+		STATKEY_PER = 1,
+		STATKEY_INT = -2,
+		STATKEY_SPD = -2,
+	) //7 - Statline - The Idea is that they're tanky and supposed to be able to block hits for a longer time, hence higher CON and END
+
+	skills = list(
+		/datum/skill/misc/swimming = 2,
+		/datum/skill/misc/climbing = 4,
+		/datum/skill/misc/sneaking = 2,
+		/datum/skill/combat/wrestling = 3,
+		/datum/skill/combat/unarmed = 2,
+		/datum/skill/combat/swords = 4,
+		/datum/skill/combat/shields = 4,
+		/datum/skill/combat/knives = 2,
+		/datum/skill/misc/reading = 1,
+		/datum/skill/misc/athletics = 4,
+		/datum/skill/misc/medicine = 2,
+		/datum/skill/labor/mathematics = 6,
+	)
+
+	traits = list(
+		TRAIT_CRITICAL_RESISTANCE,
+		TRAIT_HARDDISMEMBER,
+		TRAIT_NOPAINSTUN,
+		TRAIT_BREADY,
+		TRAIT_BLINDFIGHTING, //They use math to calculate the trajectory of attacks, so they can parry behind them, trust, ook told me
```

Comment:
why

Reply:
check comment. and check the feedback on discord.